micro_http/src/common/headers.rs

```diff
@@ -3,8 +3,12 @@
 
 use std::collections::HashMap;
 use std::io::{Error as WriteError, Write};
+use std::result::Result;
 
 use ascii::{COLON, CR, LF, SP};
+use std::cell::RefCell;
```

Comment:
The imports are separated in external dependencies (like std) and internal dependencies (like ascii and RequestError). To keep this separation you'll need to move the std imports above.